etc/brotab

```diff
@@ -9,5 +9,6 @@
 * * * * * ( sleep 30 ; /home/pi/firewalla/scripts/fire-ping.sh  >/dev/null 2>&1 )
 * * * * * ( sleep 40 ; /home/pi/firewalla/scripts/fire-ping.sh  >/dev/null 2>&1 )
 * * * * * ( sleep 50 ; /home/pi/firewalla/scripts/fire-ping.sh  >/dev/null 2>&1 )
+* * * * * ( /home/pi/firewalla/scripts/fireapi-ping.sh  >/dev/null 2>&1 )
```

Comment:
let's make it run once per two minutes to avoid overlap. Also fireapi not working is a very rare situation that we can wait for two minutes if happens.